packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -396,13 +434,18 @@ export class DeploymentManager {
     await this.waitForActiveTables(stackParams.tableNames);
   };
 
+  private printLogs = (): void => {
+    this.printer.print();
+  }
```

Comment:
does it make sense to have alias to a method just to spare `.` ? (if we have more places like this please reconsider, less code is better).